x-pack/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -43,168 +49,189 @@ function ActionMenuButton({ onClick }: { onClick: () => void }) {
   );
 }
 
+function removeEmptyItems<T extends {}>(
+  items: Array<T | null | undefined>
+): T[] {
+  return items.filter(item => !!item) as T[];
+}
+
 interface Props {
   readonly transaction: Transaction;
 }
 
-interface State {
-  readonly isOpen: boolean;
-}
+export const TransactionActionMenu: FunctionComponent<Props> = (
+  props: Props
+) => {
+  const { transaction } = props;
```

Comment:
nit: This is a matter of personal preference but what about:
```ts
export TransactionActionMenu({ transaction }: Props) {
...
}
```

There are some issues in Dev tools and jest snapshots where the component name is omitted when exporting an an (anonymous) arrow function instead of a named function.
Also, the destructuring of props seems nice in this case. 

Ofc. this is subjective and matter of style - not important stuff :)

Reply:
Do you mean `export function TransactionMenu`? Is there a way I can reproduce it? Just want to check if that is still the case.. I think Babel at least should handle this appropriately because it is a named export (but not sure).

Reply:
Yes, I missed the `function` part 😅

Reply:
It happens when using enzyme's `mount` afair. Will check again.

Reply:
I've left this for now - a `function` export is a little harder to type as a FunctionalComponent, which means that you lose some type safety (AFAIK). I'll look into the mount stuff later, could not find anything about it quickly.